layouts/one-offs/support-us.html

```diff
@@ -1,79 +1,111 @@
-    </section>
+  <section class="support-us-section">
+    <img src="{{ "./images/angelica-gifs/worm_computer.gif" | relURL }}">
+    <h2 id="web-monetization">Web Monetization</h2>
+    <p>Readers shouldn’t have to wade through intrusive ads or paywalls to read this magazine. We don’t want your data
+      nor do we want to restrict access to COMPOST. <a href="https://webmonetization.org/" target="_blank">Web
+        Monetization</a> is another way for
+      supporters like you to send us micropayments (fractions of a cent) as you view this website.</p>
+    <h3 id="how-does-it-work">How does it work?</h3>
+    <p>Web Monetization is an emerging standard for web browsers. At the moment, you need to download a
+      browser extension to make it work. We’ve found that getting a <a href="https://coil.com/signup"
+        target="_blank">Coil membership</a> is the easiest way to get started
+      with Web
+      Monetization. With a $5 monthly membership, you can also support hundreds of other Web Monetization websites like
+      <a href="https://cinnamon.video" target="_blank">Cinnamon</a>, <a href="https://gamedevjs.com/"
+        target="_blank">Gamedev.js</a>, and <a href="https://www.ladyspikemedia.com/" target="_blank">Ladyspike
+        Media</a>.</p>
+    <h3 id="setup-coil">Setup Coil</h3>
+    <ol>
+      <li>Visit <a href="https://coil.com/signup" target="_blank">https://coil.com/signup</a> and signup</li>
+      <li>Go to Settings and enter the "Membership" menu item.</li>
+      <li>Add your credit card and subscribe. It's $5 a month.</li>
+      <li>Download the extension for any browser of your choice.</li>
+    </ol>
+    <p>If everything works, you’ll see the web monetization mushroom (top right corner of the screen) animating with a
+      dollar sign, like this:</p>
+    <img src="{{ "./images/black-mushroom-webmon-active.gif" | relURL }}">
+  </section>
 
-    <!-- <section>
+  <!-- <section>
       <h1>Issue 1 Finances</h1>
       <p>As of <span todays-date></span> COMPOST Issue 1 has raised:</p>
       <h2 amount-raised></h2>
       <img src="finances-pie.png">
     </section> -->
 
-    <!-- <section>
+  <!-- <section>
       <p>Help us keep our starship in orbit!</p>
     </section>
 
     <a href="{{.Site.Home.Permalink}}">Go Home</a> -->
-    
-  </div>
+
+</div>
 {{ end }}
 
 
 {{ define "scripts" }}
-  <script>
-    var goals = JSON.parse({{ (index .Site.Data "fundraising-goals") | jsonify }})
-  </script>
-  {{ $js := resources.Get "js/contribute.js" }}
-  <script src="{{ $js.Permalink }}"></script>
-{{ end }}
+<script>
+  var goals = JSON.parse({
+    {
+      (index.Site.Data "fundraising-goals") | jsonify
+    }
```

Comment:
This is kinda wack formatting. The {{}} is template syntax. Did your editor automatically do this?

Reply:
Oof yes, that was vscode